### PR TITLE
feat: add burger menu

### PR DESCRIPTION
**Pull request by ElijahWolf1986**

### CODE REVIEW

File: .env.example
Comment (algoritmi4) [inline .env.example:1]: project constants must be stored in your .env local file, not in .env-example.

File: src/features/burgerMenu/ui/burgerMenu.tsx [hunk 1]
```diff
@@ -2,12 +2,137 @@ import { Menu } from "@headlessui/react";
...
 export function BurgerMenu() {
   return (
-    <Menu.Items>
+    <Menu.Items className="absolute flex flex-col right-0 z-[11] mt-10 w-[293px] h-[435px] px-10 py-[30px] rounded-[20px] bg-white shadow justify-start items-start gap-[15px] inline-flex focus:outline-none">
       <Menu.Item>
         {({ active }) => (
-          <a className={`${active && "bg-blue-500"}`} href="/account-settings">
-            <div className="bg-user-profile w-[24px] h-[24px]"> </div>
-            <p>Профиль</p>
+          <a className={`${active}`} href="/#">
```
Comment (algoritmi4) [inline src/features/burgerMenu/ui/burgerMenu.tsx]: you need to add array data in /lib/consts and iterate this data here with .map().

File: tailwind.config.js
Comment (algoritmi4) [inline tailwind.config.js:1]: Return it the way it was. Don`t refactor code when you developing a feature. For refactoring you can create a separate branch or at least a separate commit

File: src/features/burgerMenu/ui/burgerMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Menu } from "@headlessui/react";
+
+export function BurgerMenu() {
+  return (
+    <Menu.Items>
+      <Menu.Item>
+        {({ active }) => (
```
Comment (algoritmi4) [inline src/features/burgerMenu/ui/burgerMenu.tsx]: i`m not sure that active state need here because when navigating through pages, you do not need the active state of links

File: src/widgets/ProfileButton/ProfileButton.tsx [hunk 3]
```diff
@@ -1,10 +1,20 @@
...
 
 export function ProfileButton() {
-    let {data: profileData} = useMyDetailsQuery()
-    return (
-        <Button HTMLType='button' type='secondary' extraClass='text-base mt-2.5'>{profileData?.username}</Button>
-    )
+  let { data: profileData } = useMyDetailsQuery();
+  return (
+    <Menu>
+      {({ open }) => (
```
Comment (algoritmi4) [inline src/widgets/ProfileButton/ProfileButton.tsx]: i think you can delete this open state because headless-ui state no need to control manually, it automatically work

File: .env.example
Comment (ElijahWolf1986) [reply .env.example:1]: fixed

File: src/features/burgerMenu/ui/burgerMenu.tsx (hunk 1 above)
Comment (ElijahWolf1986) [reply src/features/burgerMenu/ui/burgerMenu.tsx]: fixed

File: src/features/burgerMenu/ui/burgerMenu.tsx (hunk 2 above)
Comment (ElijahWolf1986) [reply src/features/burgerMenu/ui/burgerMenu.tsx]: fixed

File: src/widgets/ProfileButton/ProfileButton.tsx (hunk 3 above)
Comment (ElijahWolf1986) [reply src/widgets/ProfileButton/ProfileButton.tsx]: fixed

**Review by algoritmi4 — APPROVED** [review]
